doc/procedures/jira.rst

```diff
@@ -81,30 +79,28 @@ Task Process
     * Required fields to create a task
 
         Project
-            Telescope and Site Software(TSS)
+            Data Management(DM)
```

Comment:
We should probably add something about the Team designation as well (i.e. 'Telescope and Site')